fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -243,6 +241,10 @@
                     indexingPolicy = IndexingPolicy.newBuilder()
                             .setSourceIndexSubspaceKey(sourceIndexSubspaceKey)
                             .setForbidRecordScan(origPolicy.forbidRecordScan)
+                            .setIfDisabled(origPolicy.ifDisabled)
+                            .setIfWriteOnly(origPolicy.ifWriteOnly)
+                            .setIfMismatchPrev(desiredAction)
+                            .setIfReadable(origPolicy.ifReadable)
```

Comment:
Should `IndexingPolicy` have a `.toBuilder()` method that creates a new `IndexingPolicy.Builder` with all of the previous elements set? I'm a little concerned we might otherwise forget to update the `IndexingPolicy` copying logic here (or somewhere else) and accidentally drop a field we didn't intend to.

Reply:
That'll be a very elegant way to do it.

Reply:
Done